db/helpers/users/create.js

```diff
@@ -11,7 +12,8 @@ const register = async (req, res) => {
       role,
       organizationName
     } = req.body;
-
+    const hash = bcrypt.hashSync(password, 12);
```

Comment:
Rather than having our salt in plain sight, why not make an environment variable for it? It's best to not show our hands to players with ill intent if we can help it.

Reply:
Done.